### ISSUE TITLE
extension Authremoteuser breaks maintenance/refreshLinks.php

### ISSUE THREAD

**Issue by mattias-holm** [noris-network/mediawiki-extensions-sessionprovider-remoteuser#11]
Hi,

With Authremoteuser loaded I'm unable to run the refreshLinks.php maintenance script unless I set the REMOTE_USER env. Relevant part of error message is "InvalidArgumentException from line 98 of /opt/web/mediawiki-1.28.0/extensions/Authremoteuser/AuthRemoteuser.body.php: Invalid user name"

Also, probably related to the above problem, setting a category([[category:test]] on a page does not add to the categorylinks table, why one have to run refreshLinks.php manually.

/Mattias

**Comment by fnk** [comment]
Yes, setting `REMOTE_USER` for maintenance scripts is necessary, see #7 .
I will update the `README` accordingly.

Unless you need to automate those actions via cron, you can also install the extension [Maintenance](https://www.mediawiki.org/wiki/Extension:Maintenance) and trigger the scripts via the page `Wiki/Spezial:Maintenance`.
